client/my-sites/theme/main.jsx

```diff
@@ -345,7 +345,7 @@ class ThemeSheet extends React.Component {
 		return (
 			<div className="theme__sheet-content">
 				{ config.isEnabled( 'jitms' ) && this.props.siteSlug && (
-					<AsyncLoad require="calypso/blocks/jitm" messagePathSuffix={ 'admin_notices' } />
+					<AsyncLoad require="calypso/blocks/jitm" messagePath={ 'calypso:theme:admin_notices' } />
```

Comment:
There is a conflict with #46770, should be easy to resolve (just add the `placeholder={ null }` prop).

JSX syntax nit: string props can be written as `messagePath="foo"` instead of `messagePath={ 'foo' }`.